src/time_travel/datetime_patcher.py

```diff
@@ -10,39 +12,40 @@ class DatetimePatcher(object):
     For now patching only the @today function.
     """
     
-    def __init__(self, start_time=0):
+    def __init__(self, clock):
         """Create the patch.
         
         @start_time is time in seconds since the epoch.
         """
-        self.timestamp = start_time
+        self.clock = clock
         
-        self.real_datetime = datetime_lib.datetime.fromtimestamp
+        self.real_fromtimestamp = datetime_lib.datetime.fromtimestamp
         
         self.datetime = mock.Mock(wraps=datetime_lib.datetime)
         self.datetime.mock_add_spec(
             ["today", "now", "utcnow", "fromtimestamp"])
```

Comment:
Beccause if I will not add it explicitly to the spec of the mock, and because I'm wraps, the call to {{datetime.fromtimestamp}} will raise error for not being in the spec specified for the mock.
Adding it explicitly to the mock spec, won't patch it, but rather make the call to the real classmethod (because of wraps).